python/turbine_models/custom_models/sd_inference/utils.py

```diff
@@ -0,0 +1,56 @@
+import iree.compiler as ireec
+import numpy as np
+import safetensors
+
+
+def save_external_weights(
+    mapper,
+    model,
+    external_weights=None,
+    external_weight_file=None,
+):
+    if external_weights is not None:
+        if external_weights == "safetensors":
+            mod_params = dict(model.named_parameters())
+            for name in mod_params:
+                mapper["params." + name] = name
+            if external_weight_file:
+                safetensors.torch.save_file(mod_params, external_weight_file)
+                print("Saved params to", external_weight_file)
+
+
+def largest_error(array1, array2):
+    absolute_diff = np.abs(array1 - array2)
+    max_error = np.max(absolute_diff)
+    return max_error
+
+
+def compile_to_vmfb(module_str, target_backends, safe_name):
```

Comment:
missing the different backend support that was recently added to stateless_llama

Reply:
added the new backend support